tests/core/pyspec/eth2spec/utils/bls.py

```diff
@@ -68,7 +105,10 @@ def entry(*args, **kw):
 @only_with_bls(alt_return=True)
 def Verify(PK, message, signature):
     try:
-        result = bls.Verify(PK, message, signature)
+        if bls == arkworks_bls:  # no signature API in arkworks
```

Comment:
We could monkey-patch arkworks_bls with the py_ecc functionality where it is lacking instead of doing all these if/else statements. I guess the benefit here is to make the foot-gun explicit, but it might be nice to not have all of these extra statements

Reply:
I'm fine with explicit. less foot-gunny